src/en.json

```diff
@@ -14,8 +14,8 @@
             "AccountIsShown": "Account is shown"
         },
         "assets": {
-            "addressCopied": "Wallet address copied to clipboard",
-            "copyAddressHint": "Copy the wallet address",
+            "addressCopied": "Profile address copied to clipboard",
+            "copyAddressHint": "Copy the profile address",
```

Comment:
@AnnStepanova could you please review all changes in English localization and tell whether all of them are correct or we first need to improve some of them and only after that make translations for Korean and Russian locales?